.github/workflows/release.yml

```diff
@@ -159,33 +167,33 @@ jobs:
       - name: Download Artifact
         uses: actions/download-artifact@v3
         with:
-          name: manylinux-wheel
-          path: manylinux-wheel
+          name: ${{ needs.wait-for-build-test.outputs.manylinux-x86-wheel-artifact-name }}
```

Comment:
Why not simply replace this name: ${{ ...} with the actual value we know it contains? It's a constant!
What's the gain of using this very long variable name, than even contain this ...manylinux-x86-wheel.... in it
with simply manylinux-x86-wheel!
It's a shorter, and it removes the need for those ENV:, for those needs.wait.xxx everywhere, for those echo ... >> $GITHUB_OUTPUT.